packages/config/config/devices/0x0258/nas-pd01z.json

```diff
@@ -0,0 +1,173 @@
+			"valueSize": 2,
+			"minValue": 5,
+			"maxValue": 600,
+			"defaultValue": 30
+		},
+		"3": {
+			"label": "Basic Set Level",
+			"description": "Basic Set Command will be sent when motion is triggered",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 99,
+			"unsigned": true
+		},
+		"4": {
+			"label": "PIR Detector",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 255
+				}
+			]
+		},
+		"5": {
+			"label": "Light Sensor Brightness Threshold",
+			"description": "The light sensor is active when the brightness is above this level",
+			"unit": "lux",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 100
+		},
+		"6": {
+			"label": "PIR Re-trigger Interval",
+			"description": "Must be less than the On/Off Duration",
+			"unit": "seconds",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 8,
+			"defaultValue": 8
+		},
+		"7": {
+			"label": "Light Sensor Polling Interval",
+			"description": "The Interval after which the Light Sensor Measure is polled.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 60,
+			"maxValue": 36000,
+			"defaultValue": 180,
+			"unsigned": true
+		},
+		"8": {
+			"label": "Lux Level Function",
+			"description": "When enabled, send Basic Set if brighness falls below threshold.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"9": {
+			"label": "Lux Level Changed Report",
+			"description": "How much the Lux Level must change",
+			"unit": "lux",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 100,
+			"unsigned": true
+		},
+		"10": {
+			"label": "Temperature Differential Report Threshold",
+			"unit": "0.1 °C",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 5
+		},
+		"11": {
+			"label": "LED: Blink on Motion",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"12": {
+			"label": "Motion Event Report One Time Enable",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"99": {
+			"label": "Ambient Light Intensity Calibration",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 65535,
+			"defaultValue": 10,
+			"unsigned": true
```

Comment:
Does this parameter have a unit?

Reply:
It seems it uses a calibration formula, so the unit represents k = Vm / Vs.

There isn't much info in the user manual about how to calibrate exactly, but I found this online:

99. Ambient light intensity calibration
This parameter defines the calibrated scale for ambient light intensity. Because the
method and position that the sensor mounted and the cover of sensor will bring
measurement error, user can get more real light intensity by this parameter setting. User
should run the steps as blows for calibrating
1) Set this parameter value to default(Assumes the sensor has been added in a Z‐Wave
Network).
2) Place a digital luxmeter close to sensor and keep the same direction, monitor the light
intensity value (Vm) which report to controller and record it. The same time user should
record the value (Vs) of luxmeter.
3) The scale calibration formula: k = Vm / Vs.
4) The value of k is then multiplied by 1000 and rounded to the nearest whole number.
5) Set the value getting in 5) to this parameter, calibrate finished.
For example, Vm = 300, Vs = 2600, then
k = 300 / 2600 = 0.11538
k = 0.11538 * 1000 = 115.38 ≈ 115
The parameter should be set to 115.
Function: Ambient light intensity calibrate

Reply:
Okay, let's keep it as is :D